functions/docs.js

```diff
@@ -13,9 +15,9 @@ fetch(`https://djsdocs.sorta.moe/v2/embed?src=${branch}&q=${encodeURIComponent(q
       .then(res => res.json())
       .then(json => {
         if (!json) return message.chanel.send(`:x: | No results for ${query}`)
-        message.channel.send({ embed: json });
+       return message.channel.send({ embed: json });
       })
       .catch(() => {
-        message.channel.send(`:x: | An unexpected err occrued`)
+        return message.channel.send(`:x: | An unexpected err accrued`)
```

Comment:
Should be `occurred`, both were wrong.

Reply:
What, I remember updating it? Well, yea. I'll fix it